langchain/__init__.py

```diff
@@ -45,7 +45,7 @@
 from langchain.utilities.google_serper import GoogleSerperAPIWrapper
 from langchain.utilities.searx_search import SearxSearchWrapper
 from langchain.utilities.wolfram_alpha import WolframAlphaAPIWrapper
-from langchain.vectorstores import FAISS, ElasticVectorSearch
+from langchain.vectorstores import FAISS, ElasticVectorSearch, OpenSearchVectorSearch
```

Comment:
lets not do this, trying to encourage putting things at the module level

Reply:
ack